manifests/init.pp

```diff
@@ -124,7 +134,7 @@
     owner   => 'root',
     group   => 'root',
     mode    => '0640',
-    content => template("${module_name}/etc/sysconfig/autofs.erb"),
+    content => template("autofs/etc/sysconfig/autofs.erb"),
```

Comment:
It's not a big deal and doesn't affect my approval, but―as a practice―I prefer the ``template("${module_name}/[...].erb")`` approach because it keeps the code portable between module name changes.